packages/admin-ui-docs/content/table/code.mdx

```diff
@@ -210,13 +217,17 @@ function Example() {
 render(<Example />)
 ```
 
-### Resolvers
+</Section>
+
+<Section hasExplanation>
+
+## Resolvers
 
 Resolvers are rendering functions that target a specific data type. The main usage is to render the same data types consistently along with admin applications.
 
-#### Key concepts
+<Section hasExplanation>
```

Comment:
Nope, it will be something like:
```html
<div>
  <h2>
    Title
   </h2>
   <div>
      <h3>...</h3>
      <h3>...</h3>
   </div>
 </div>
```